### DISCUSSION TITLE
Create a link to Curb Data Specification (CDS) feeds

### DISCUSSION THREAD

**Discussion by schnuerle** [usdot-jpo-ode/jpo-wzdx#323] [Ideas] [answered]
While developing the OMF's [Curb Data Specification](https://github.com/openmobilityfoundation/curb-data-specification) (CDS) [a discussion](https://github.com/openmobilityfoundation/curb-data-specification/discussions/53) was had how to connect CDS and WZDx public feeds in each spec via URLs and IDs. The idea is that work zones could impact usage and access to curb space, and each spec could reference the other in these cases so road users would have a full picture of how their usage could be affected and for safety.

We eventually moved this to an [Issue](https://github.com/openmobilityfoundation/curb-data-specification/issues/74) which had some support from Omaha and Seattle and others during some public working group meetings, and now we have a [Pull Request](https://github.com/openmobilityfoundation/curb-data-specification/pull/96) (which we'd like your feedback on) for how to reference public WZDx feeds from CDS. What we'd like to see is also a way to reference public CDS feeds from WZDx.

**Comment by j-d-b** [comment, marked as answer]
This will be resolved by #326.
